R/tools.R

```diff
@@ -30,3 +30,100 @@ generate_examples <- function(function_name, args){
   print(knitr::kable(ret, format="markdown"))
   ret
 }
+
+#' Check package update from cran and requiredRpackage json
```

Comment:
I think this function is about our internal process and should not be shared to public.